devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusBuild.java

```diff
@@ -78,13 +78,18 @@ public FileCollection getClasspath() {
     }
 
     @OutputFile
-    public File getOutputDir() {
+    public File runnerJar() {
         return new File(getProject().getBuildDir(), extension().finalName() + "-runner.jar");
     }
 
+    @OutputFile
+    public File fastJar() {
+        return new File(getProject().getBuildDir(), "quarkus-app");
```

Comment:
It's not a problem if only one of the two is created when running the goal?

Reply:
I had the same question myself, so I ran multiple tests and didn't see any problem.

Of course this will need to vetted by our Gradle experts :)

Reply:
Actually I had the same thought too, there are tests checking, if this is ok, we could also declare the native build result.

Reply:
@glefloch yeah, that's a good idea for a follow up

Reply:
Oh this should be `@OutputDirectory` then.
 And reading the documentation, it should works as expected: 
```
This will cause the task to be considered out-of-date when the directory path or task output to that directory has been modified since the task was last run.
```
Missing directory should not be a problem.

Reply:
Oh nice, I didn't know of that annotation